recipes/aihwkit/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set pname = "aihwkit" if cuda_compiler_version == "None" else "aihwkit-gpu" %}
+{% set name = "aihwkit" %}
+{% set version = "0.8.0" %}
+{% set number = 0 %}
+
+package:
+  name: {{ pname|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1cfd8ca9fec90c7df1d0eacefbc5b421008bbf8cefda481793cd9e3d75710ae7
+
+build:
+  skip: true  # [win or osx]
+  skip: true  # [cuda_compiler_version in ("10.2", "11.0", "11.1")]
+  missing_dso_whitelist:
+    - $RPATH/libtorch_python.so
+    - lib/libgomp.so.1
+  number: {{ number }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != 'None']
+    - cmake
+    - numpy
+    - pybind11
+    - make
```

Comment:
```suggestion
    - make
    - libgomp   # [linux]
```

Reply:
I made the suggested change. Would you please review the PR?  thanks.